feed_test.go

```diff
@@ -84,3 +89,84 @@ func TestNotificationFeedBasic(t *testing.T) {
 		t.Fatal()
 	}
 }
+
+func TestUpdateActivities(t *testing.T) {
```

Comment:
Test should go int `feed_flat_test.go` with all the other integration tests and it should use helpers such as `PreTestSetup` and `getFlatFeed`.

Reply:
this is not a flat feed-only test. it's testing a method of `baseFeed`, therefore should be tested where the other `baseFeed` methods are tested. I agree on the `PreTestSetup`, which should then be moved to a separate helper package.

Reply:
actually, makes more sense to split the test in two and put both into flat feed tests and aggregated feeds tests ;)